CONTRIBUTING.md

```diff
@@ -96,9 +97,67 @@ Ensure all the following conditions are met (on top of you agreeing with the cha
 
 To protect the Hyperledger Cactus source code, GitHub pull requests are accepted from forked repositories only. There are also quality standards identified and documented here that will be enhanced over time.
 
-
-
-
+### Create local branch
+
+> Whenever you begin work on a new feature or bugfix, it's important that you create a new branch. 
+
+1. Clone your fork to your local machine
+2. Setup your local fork to keep up-to-date (optional)
+   ```
+   # Add 'upstream' repo to list of remotes
+   git remote add upstream https://github.com/hyperledger/cactus.git
+
+   # Verify the new remote named 'upstream'
+   git remote -v
+
+   # Fetch from upstream remote
+   git fetch upstream
+
+   # View all branches, including those from upstream
+   git branch -va
+
+   # Checkout your master branch and merge upstream. 
+   # Run those commands whenever you want to synchronize with master branch
+   git checkout master
+   git merge upstream/master
```

Comment:
Would it be okay to encourage people to rebase here instead? They'll have to do that eventually anyway at the time of the PR submission so probably less confusing if they follow that pattern to begin with, like 

`git rebase upstream/master`